### ISSUE TITLE
Node user should be supplied

### ISSUE THREAD

**Issue by matthewvalimaki** [smebberson/docker-alpine#12]
alpine-consul-nginx-nodejs and alpine-nginx-nodejs should provide nodejs user. user-consul-nginx-nodejs and user-nginx-nodejs then both use this user in s6 run scripts. Right now node server is running as root.

**Comment by smebberson** [comment]
@matthewvalimaki, true! Not a good scenario to have them running as root. I'll get this added.

**Comment by matthewvalimaki** [comment]
See pull request #33 for resolution.
